presto-iceberg/src/test/java/com/facebook/presto/iceberg/TestIcebergSmoke.java

```diff
@@ -418,6 +425,47 @@ public void testInsertIntoNotNullColumn()
         // TODO: To support non-null column. (NOT_NULL_COLUMN_CONSTRAINT)
     }
 
+    @Test
+    public void testConcurrentInsert()
+    {
+        final Session session = getSession();
+        assertUpdate(session, "CREATE TABLE test_concurrent_insert (col0 INTEGER, col1 VARCHAR) WITH (format = 'ORC')");
+
+        int concurrency = 5;
+        final String[] strings = {"one", "two", "three", "four", "five"};
+        final CountDownLatch countDownLatch = new CountDownLatch(concurrency);
+        AtomicInteger value = new AtomicInteger(0);
+        List<Thread> threads = Stream.generate(() -> new Thread(() -> {
+            int i = value.getAndIncrement();
+            try {
+                assertUpdate(session, format("INSERT INTO test_concurrent_insert VALUES(%s, '%s')", i + 1, strings[i]), 1);
```

Comment:
will junit failed and collect the test result for new threads?  I think the junit only takes care of the assert in the main thread -- I might be wrong, could you confirm?

Reply:
Thanks for review! I am not sure whether the junit assertion works in a non main thread. I only want to imitate the concurrent insertion here, but I don't know how to insert data without a assertion, so I placed a `assertUpdate` here... I'll find better way to improve it.

Reply:
I did some change, can you have a look again?